index.html

```diff
@@ -2179,8 +2179,8 @@ <h4 class="modal-title" id="myModalLabel">Long Vesion Code of Conduct</h4>
                <!-- Social Icons -->
                <div class="footer-social-icons">
                    <a href="http://bit.ly/pydelhi-git" target="_blank"><i class="fa-2x fa fa-github"></i></a>
-                   <a href="http://bit.ly/pydelhi-fb" target="_blank"><i class="fa-2x fa fa-facebook"></i></a>
-                   <a href="http://bit.ly/pydelhi-twitter" target="_blank"><i class="fa-2x fa fa-twitter"></i></a>
+                   <a href="http://bit.ly/pydelhiconf_fb" target="_blank"><i class="fa-2x fa fa-facebook"></i></a>
```

Comment:
Can you add a comment here, that you own the analytics for these links. Rest looks good.